app/main/controllers/template/RTMediaAJAX.php

```diff
@@ -1,37 +1,49 @@
 <?php
-
-/*
- * To change this template, choose Tools | Templates
- * and open the template in the editor.
+/**
+ * Handle media ajax.
+ *
+ * @package rtMedia
  */
 
 /**
- * Description of RTMediaAJAX
+ * Class to handle media ajax.
  *
  * @author udit
  */
 class RTMediaAJAX {
 
+	/**
+	 * RTMediaAJAX constructor.
+	 */
 	public function __construct() {
 		add_action( 'wp_ajax_rtmedia_backbone_template', array( $this, 'backbone_template' ) );
 		add_action( 'wp_ajax_rtmedia_create_album', array( $this, 'create_album' ) );
 	}
 
-	function backbone_template() {
+	/**
+	 * Display media gallery items template.
+	 */
+	public function backbone_template() {
 		include RTMEDIA_PATH . 'templates/media/media-gallery-item.php';
 	}
 
-	function create_album() {
-		$nonce = filter_input( INPUT_POST, 'create_album_nonce', FILTER_SANITIZE_STRING );
-		$_name = filter_input( INPUT_POST, 'name', FILTER_SANITIZE_STRING );
+	/**
+	 * Create album.
+	 */
+	public function create_album() {
+		$nonce        = filter_input( INPUT_POST, 'create_album_nonce', FILTER_SANITIZE_STRING );
+		$_name        = filter_input( INPUT_POST, 'name', FILTER_SANITIZE_STRING );
 		$_description = filter_input( INPUT_POST, 'description', FILTER_SANITIZE_STRING );
```

Comment:
@vaishaliagola27 Sanitize it using `sanitize_text_field()` function if see `FILTER_SANITIZE_STRING` in filter_input() function. Do same at other places as well.

`$_description = sanitize_text_field( filter_input( INPUT_POST, 'description', FILTER_SANITIZE_STRING ) );`

Reply:
I'll Address this feedback in new PR for the same files.

Reply:
@chandrapatel Please check. I've addressed this feedback in same PR.